packages/client/src/subscribe/SubscriptionSession.ts

```diff
@@ -100,7 +101,7 @@ export class SubscriptionSession {
     private async subscribe(): Promise<NetworkNodeStub> {
         const node = await this.node.getNode()
         node.addMessageListener(this.onMessageInput)
-        await node.subscribe(this.streamPartId)
+        node.subscribe(this.streamPartId)
```

Comment:
Was the removal of `await` intentional? (commit 9fcd5e8a084cff65a2aaf0682e20b4ef85e6d565)